### PR TITLE
column width < 80

### PR DESCRIPTION
**Pull request by vpj**

### CODE REVIEW

File: pytorch/mnist/plain.py [hunk 1]
```diff
@@ -59,8 +57,9 @@ def test(epoch, model, test_loader, device, summary_writer):
             data, target = data.to(device), target.to(device)
 
             output = model(data)
-            test_loss += F.nll_loss(output, target, reduction='sum').item()  # sum up batch loss
-            pred = output.argmax(dim=1, keepdim=True)  # get the index of the max log-probability
+            test_loss += F.cross_entropy(output, target,
```
Comment (vpj) [inline pytorch/mnist/plain.py:60]: It has to be `cross_entropy` right?

Comment (hnipun) [reply pytorch/mnist/plain.py:60]: https://pytorch.org/docs/stable/nn.functional.html#nll-loss also works I think.
